robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -379,7 +377,7 @@ def validate_current_and_next_year_values(self, amount, rdtype=None):
             if rdtype == "Open":
                 next_year_value = next_year_value + (12-next_year_count)*int(amount)
             values['Current Year Value']=f"${curr_year_value}.00"
-            values['Next Year Value']=f"${ next_year_value}.00"
+            #values['Next Year Value']=f"${ next_year_value}.00"
```

Comment:
Did u intentionally comment it out ?

Reply:
Yes this validation does not apply anymore

Reply:
please remove this if not needed